README.md

```diff
@@ -237,5 +238,8 @@ docker-compose build utils
 
 # To run tests
 docker-compose -f docker-compose.ci.yml run utils poetry run pytest
+
+# To autoformat
+docker-compose run utils poetry run autopep8 --in-place --aggressive --aggressive --recursive .
```

Comment:
is this supposed to be extra-aggressive?

Reply:
yea - it was in the autopep8 example code. Figured it'd be harmless and extra formatted